### ISSUE TITLE
No implicit UrlForm EntityEncoder.  Why?

### ISSUE THREAD

**Issue by rossabaker** [http4s/http4s#288]
Is there a good reason we did not make this implicit, with an implicit charset?  We use this pattern elsewhere (i.e., stringEncoder).

```scala
  def entityEncoder(charset: Charset): EntityEncoder[UrlForm] =
    EntityEncoder.stringEncoder(charset)
      .contramap[UrlForm](encodeString(charset))
      .withContentType(`Content-Type`(MediaType.`application/x-www-form-urlencoded`, charset))
```

/cc @adelbertc

**Comment by bryce-anderson** [comment]
I can't think of one.

**Comment by bryce-anderson** [comment]
Closed by f85d947e8fec24f37d5a3ed030a54a1659c04354.
